pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/ValidatedJsonInputRoute.java

```diff
@@ -140,9 +145,31 @@ private T handleJsonUnmarshall(Request request, Response response) {
         } catch (JsonSyntaxException e) {
             log.warn("JSON payload could not be converted to object of class: " + getTargetClass(request).getName(), e);
             ApiError err = new ApiError(ErrorCodes.BAD_PAYLOAD, "Request payload could not be parsed and converted to expected type");
-            ResponseUtil.haltError(response, getUnmarshallErrorStatus(), err);
-            return null;
+            throw ResponseUtil.haltError(response, getUnmarshallErrorStatus(), err);
+        } catch (JsonParseException exception) {
+            var cause = Optional.ofNullable(exception.getCause()).orElse(exception);
+
+            /*
+             * See if there's a better way of handling these errors. The JsonParseException _works_
+             * but Google's documentation seems to indicate these are intended more for errors in the
+             * structure of the JSON, and not errors in the content.
+             * 
+             * 2022.07.13 - bskinner
+             */
+            if (cause instanceof DateTimeParseException) {
+                // Likely caused by an adapter for a LocalDate or a LocalDateTime
+                // failing due to a poorly formatted date string
+                log.debug("failed to unmarshall json object: a date-typed property value does not match the required format", cause);
```

Comment:
Let's raise log level to warn or error. Likely that if this happens is because developer does not have things just right and they could be looking in logs for clues.